features/beacon.yml

```
@@ -0,0 +1,5 @@
+name: Beacon
+description: The `navigator.sendBeacon()` method asynchronously sends an HTTP POST request containing a small amount of data to a web server.
```

This is too long, but I think gets at some of the key facts we ought to mention here.

```suggestion
description: The `navigator.sendBeacon()` method asynchronously sends an HTTP POST request with a limited amount of data, typically for application state, events, and analytics data. The one-way beacon is guaranteed to be initiated before the browser unloads the page, but the sender does not get to handle the response.
```